specificatie/openapi.yaml

```diff
@@ -1,173 +1,152 @@
+          description: |
+                        Een aanduiding die aangeeft dat gegevens wel of niet verstrektmogen worden. Indien true: op verzoek van deze persoon is het verstrekken van gegevens over deze persoon aan bepaalde derden niet toegestaan.
         inOnderzoek:
           $ref: "#/components/schemas/ReisdocumentInOnderzoek"
     ReisdocumentHal:
       allOf:
         - $ref: '#/components/schemas/Reisdocument'
         - properties:
             _links:
-              $ref: "#/components/schemas/Reisdocument_links"
-    Reisdocument_links:
+              $ref: "#/components/schemas/ReisdocumentLinks"
+    ReisdocumentLinks:
       type: "object"
       properties:
         self:
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.1.0/api-specificatie/common.yaml#/components/schemas/HalLink"
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/HalLink"
         ingeschrevenPersoon:
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.1.0/api-specificatie/common.yaml#/components/schemas/HalLink"
-    AanduidingInhoudingVermissingReisdocument_enum:
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.2.0/api-specificatie/common.yaml#/components/schemas/HalLink"
+    AanduidingInhoudingVermissingReisdocumentEnum:
       type: "string"
-      description: "De mogelijke waarden van de aanduiding van inhouding of vermissing\
-        \ van een Nederlands reisdocument. Zie logisch ontwerp BRP bij de stamtabellen:\n\
-        * `ingehouden_of_ingeleverd` - I\n* `vermist` - V\n* `rechtswege` - R\n* `niet_in_bezit_van`\
-        \ - N"
+      description: |
+                    De mogelijke waarden van de aanduiding van inhouding of vermissing van een Nederlands reisdocument. Zie logisch ontwerp BRP bij de stamtabellen:
+                    * `ingehouden_of_ingeleverd` - I
+                    * `vermist` - V
+                    * `rechtswege` - R
+                    * `niet_in_bezit_van` - N"
       enum:
       - "ingehouden_of_ingeleverd"
       - "vermist"
       - "rechtswege"
       - "niet_in_bezit_van"
     ReisdocumentInOnderzoek:
       type: "object"
-      description: "Een groep van booleans om aan te geven welke gegevens van het\
-        \ reisdocument in onderzoek zijn. Als de hele categorie in onderzoek is worden\
-        \ alle rubrieken binnen die categorie op _true_ gezet. Zie de [functionele\
-        \ specificaties](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/blob/master/features/in_onderzoek.feature)\
-        \ ."
+      description:  |
+                      Geeft aan welke gegevens over de naam in onderzoek zijn. Zie de [functionele specificaties](https://github.com/VNG-Realisatie/Haal-Centraal-Reisdocumenten-bevragen/blob/v1.0.0/features/in_onderzoek.feature)
```

Comment:
"Geeft aan welke gegevens over de **naam** in onderzoek"

Gaat niet over de naam, maar over het reisdocument.

Voorstel: "Geeft aan welke gegevens in onderzoek zijn."

Reply:
Aangepast